examples/bpk-component-bottom-sheet/examples.tsx

```diff
@@ -178,6 +178,26 @@ const OverflowingExample = () => (
   </BottomSheetContainer>
 );
 
+const WrapHeaderTextExample = () => (
```

Comment:
Demo the issue
![image](https://github.com/Skyscanner/backpack/assets/3943125/3643d851-4b1c-4ea0-bfd9-72074914ec5a)